ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,251 @@
+The subprocess to be started is defined in the configuration.
+
+### Configuration
+
+- *Workflow Key*   
+  Enter the workflow key of the workflow that will be started as a subprocess.
+
+
+
+## Switch case action
+
+![Switch case action](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/SwitchCaseAction.png "[Switch case action]")
+
+The *Switch case action* core action is used to compare the input value with up to 6 criteria and output this value via a different branch for each case. Additionally, this core action enables to output the input value via a separate branch if the input value does not match any of the specified cases or to output the input value without any comparison by a separate branch.    
+The data runs via the *in* input port into the workflow action and can be output via each connected output port. If the *origin* output port is connected, the input value is always output via this port without any further action. Additionally, if one or several of the *case1* to *case6* output ports are connected, the input value is compared with the criterion specified in the respective case and output via the output port of the case where the criterion matches the input value. If no criterion of the cases matches the input value, the input value is output via the *default* output port, or, if the *default* output port is not connected, the action fails.    
+The criteria to be compared with are defined in the configuration. Further, you can set the comparison to be stopped after a match.
+
+[comment]: <> (Wird der default value nur ausgegeben, wenn auch der origin port nicht verbunden ist?)
+
+### Configuration  
+
+- *Path*   
+  Enter the path to the property of the input object to be compared with.
+
+  > [Info] The properties of an entity you can refer to are described in the relevant API documentation.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Stop on match*   
+  Enable this toggle to stop comparing the input value with the specified cases once the input value matches the first case. Disable the toggle to compare the input value with all specified cases, regardless of whether a case matches or not. By default, this toggle is disabled.
+
+- *Case X*    
+  Enter the value to be compared with the input value. The value must be valid JSON. You can enter a single value or an array with multiple values. If you enter single value, it is checked if the input value equals the specified value. If you enter an array, it is checked if the input value equals one of the values in the array.   
+  You can specify up to 6 different cases. The cases are checked in order, that means, first *Case 1*, then *Case 2*, etc.
+
+[comment]: <> (sind die values in den Feldern auch type strict?)
```

Comment:
in welchen feldern? beim compare-Wert muss man json angeben, das ist sowieso immer typenbehaftet.

Reply:
vgl. Wait for criterion